test/cat.js

```diff
@@ -64,3 +64,38 @@ test('glob', t => {
   t.truthy(result.search('test1') > -1); // file order might be random
   t.truthy(result.search('test2') > -1);
 });
+
+test('simple with numbers', t => {
+  const result = shell.cat('-n', 'test/resources/cat/file1');
+  t.falsy(shell.error());
+  t.is(result.code, 0);
+  t.is(result.toString(), '     1 test1\n');
+});
+
+test('multiple with numbers', t => {
+  const result = shell.cat('-n', 'test/resources/cat/file2', 'test/resources/cat/file1');
+  t.falsy(shell.error());
+  t.is(result.code, 0);
+  t.is(result.toString(), '     1 test2\n     2 test1\n');
+});
+
+test('simple without EOF', t => {
+  const result = shell.cat('test/resources/cat/file3');
+  t.falsy(shell.error());
+  t.is(result.code, 0);
+  t.is(result.toString(), 'test3');
+});
+
+test('simple numbers without EOF', t => {
+  const result = shell.cat('-n', 'test/resources/cat/file3');
+  t.falsy(shell.error());
+  t.is(result.code, 0);
+  t.is(result.toString(), '     1 test3');
+});
+
+test('multiple numbers without EOF', t => {
```

Comment:
Can we add a test to check a file with 10+ lines (just to check that padding works as expected)?